indico/web/client/styles/modules/_category.scss

```diff
@@ -26,13 +26,32 @@
 }
 
 .category-container {
-  font-family: 'Helvetica Neue', Helvetica, Verdana, Arial, sans-serif;
```

Comment:
Wow, this nonsense setting a font not used anywhere else was there for 11 years and nobody noticed... thanks for removing it! :D (it also made me spot some more legacy CSS in this file, such as the `li h3` which has been completely unused for 11 years as well)